GW2EIEvtcParser/EIData/ProfHelpers/ProfHelper.cs

```diff
@@ -25,6 +25,46 @@ internal static class ProfHelper
             new BuffGainCastFinder(PortalEntranceWhiteMantleWatchwork, PortalWeavingWhiteMantleWatchwork),
             new BuffGainCastFinder(PortalExitWhiteMantleWatchwork, PortalUsesWhiteMantleWatchwork).UsingBeforeWeaponSwap(true),
+            // Relics
+            new BuffGainCastFinder(RelicOfVass, RelicOfVass).UsingOrigin(InstantCastFinder.InstantCastOrigin.Gear),
+            new BuffGainCastFinder(RelicOfTheFirebrand, RelicOfTheFirebrand).UsingOrigin(InstantCastFinder.InstantCastOrigin.Gear),
+            new BuffGainCastFinder(RelicOfIsgarrenTargetBuff, RelicTargetPlayerBuff).UsingChecker((bae, combatData, agentData, skillData) =>
+            {
+                return combatData.GetBuffData(RelicOfIsgarrenTargetBuff).Where(x => x.CreditedBy == bae.To && Math.Abs(x.Time - bae.Time) < ServerDelayConstant).Any();
+            }).UsingOrigin(InstantCastFinder.InstantCastOrigin.Gear),
+
+            new BuffGainCastFinder(RelicOfPeithaTargetBuff, RelicTargetPlayerBuff).UsingChecker((bae, combatData, agentData, skillData) =>
+            {
+                return combatData.GetBuffData(RelicOfPeithaTargetBuff).Where(x => x.CreditedBy == bae.To && Math.Abs(x.Time - bae.Time) < ServerDelayConstant).Any();
+            }).UsingOrigin(InstantCastFinder.InstantCastOrigin.Gear),
+
+            new BuffGainCastFinder(RelicOfTheDragonhunterTargetBuff, RelicTargetPlayerBuff).UsingChecker((bae, combatData, agentData, skillData) =>
+            {
+                return combatData.GetBuffData(RelicOfTheDragonhunterTargetBuff).Where(x => x.CreditedBy == bae.To && Math.Abs(x.Time - bae.Time) < ServerDelayConstant).Any();
+            }).UsingOrigin(InstantCastFinder.InstantCastOrigin.Gear),
+            new EffectCastFinder(RelicOfCerusHit, EffectGUIDs.RelicOfCerusEye).UsingOrigin(InstantCastFinder.InstantCastOrigin.Gear),
+            new EffectCastFinder(RelicOfTheIce, EffectGUIDs.RelicOfIce).UsingICD(1000).UsingOrigin(InstantCastFinder.InstantCastOrigin.Gear),
+            new EffectCastFinder(RelicOfFireworks, EffectGUIDs.RelicOfFireworks).UsingOrigin(InstantCastFinder.InstantCastOrigin.Gear),
+            new EffectCastFinder(RelicOfTheCitadel, EffectGUIDs.RelicWhiteCircle).UsingChecker((evt, combatData, agentData, skillData) =>
+            {
+                combatData.TryGetEffectEventsByGUID(EffectGUIDs.RelicOfTheCitadelExplosion, out IReadOnlyList<EffectEvent> effects);
+                return effects != null && effects.Any(x => x.Time > evt.Time);
+            }).UsingOrigin(InstantCastFinder.InstantCastOrigin.Gear),
+            new EffectCastFinder(RelicOfTheNightmare, EffectGUIDs.RelicWhiteCircle).UsingChecker((evt, combatData, agentData, skillData) =>
+            {
+                combatData.TryGetEffectEventsByGUID(EffectGUIDs.RelicOfTheNightmare, out IReadOnlyList<EffectEvent> effects);
+                return effects != null && effects.Any(x => x.Time > evt.Time);
```

Comment:
Not sure to get that condition, isn't that too permissive? Not Src/Dst check either, what happens if I'm running a relic triggering WhiteCircle and someone else also does it? Wouldn't it multi trigger, for ex:
I run Nightmare, squad mate run Krait, they trigger Krait at time N (so white + krait effect), I trigger Nightmare at time M such as M < N, wouldn't I also register a Krait activation (I activated white and the condition where Krait effect exist with a higher timestamp exists)

Reply:
the only secondary effect that has a src is the wizard tower, all the others don't have one

Reply:
I'm guessing the extra effects does not happen instantly?

Reply:
no they have different delays since the white circles is there as a warning

Reply:
In that case they should be commented until we find a better way to detect them, unfortunately the current implementation collides with each other